site/docs/guidelines/spacing.mdx

```diff
@@ -18,15 +18,15 @@ Spacing throughout our design system utilizes ‘Grid Units’ to inform spacing
 
 | Grid Units | Pixels | Rem | Code |
 | ---------- | ------ | --- | ---- |
-| 0.25 (Minimum spacing) | 6px | 0.375rem | $ca-grid / 4 |
-| 0.5 | 12px | 0.75rem | $ca-grid / 2 |
-| 1 | 24px | 1.5rem | $ca-grid |
-| 1.5 | 36px | 2.25rem | $ca-grid * 1.5 |
-| 2 | 48px | 3rem | $ca-grid * 2 |
-| 2.5 | 60px | 3.75rem | $ca-grid * 2.5 |
-| 3 | 72px | 4.5rem | $ca-grid * 3 |
-| 3.5 | 84px | 5.25rem | $ca-grid * 3.5 |
-| 4 | 96px | 6rem | $ca-grid * 4 |
+| 0.25 (Minimum spacing) | 6px | 0.375rem | $kz-spacing-xs | 
+| 0.5 | 12px | 0.75rem | $kz-spacing-sm | 
+| 1 | 24px | 1.5rem | $kz-spacing-md |
+| 1.5 | 36px | 2.25rem | $kz-spacing-lg | 
+| 2 | 48px | 3rem | $kz-spacing-xl | 
+| 2.5 | 60px | 3.75rem | $kz-spacing-xxl | 
+| 3 | 72px | 4.5rem | $kz-spacing-xxxl | 
+| 3.5 | 84px | 5.25rem | $kz-spacing-xxxxl | 
+| 4 | 96px | 6rem | $kz-spacing-xxxxxl | 
```

Comment:
If you wanted to get real fancy, you might align the pipes so it's readable as a table in Markdown